### PR TITLE
feat(idux): add scope supports

### PR DESCRIPTION
**Pull request by fengxiaodong28**

### CODE REVIEW

File: src/core/resolvers/idux.ts [hunk 1]
```diff
@@ -33,6 +33,12 @@ export interface IduxResolverOptions {
    * @default 'default'
    */
   importStyleTheme?: string
+
+  /**
+   * The alias of '@idux` package.
+   * @default '@idux'
+   */
+   alias?: string
```
Comment (sxzz) [inline src/core/resolvers/idux.ts:41]:
```suggestion
   packageName?: string
```

Comment (fengxiaodong28) [reply src/core/resolvers/idux.ts:41]: `${alias}/${packageName}/${dirname}`

Comment (sxzz) [reply src/core/resolvers/idux.ts:41]: I see, how about `scope`

```suggestion
   scope?: string
```

Comment (fengxiaodong28) [reply src/core/resolvers/idux.ts:41]: ok，to the scope

Comment (fengxiaodong28) [reply src/core/resolvers/idux.ts]: @sxzz is ok

**Comment by fengxiaodong28** [comment]
@sxzz please publish packages，thank you

**Comment by fengxiaodong28** [comment]
@antfu

**Pull request merged**